packages/angular_devkit/build_angular/src/angular-cli-files/plugins/any-component-style-budget-checker.ts

```diff
@@ -33,7 +33,7 @@ export class AnyComponentStyleBudgetChecker implements Plugin {
         }
 
         const componentStyles = Object.keys(compilation.assets)
-            .filter((name) => name.endsWith('.css'))
+            .filter((name) => ['.css', '.scss', '.less', '.styl'].some((styleExtension) => name.endsWith(styleExtension)))
```

Comment:
`.sass` is missing from the extension list, also this logic can be simplified slightly as there is no need to re-declare the list of styles extensions for every iteration.

```ts

const cssExtensions = [
  '.css',
  '.scss',
  '.less',
  '.styl',
  '.sass',
];

const componentStyles = Object.keys(compilation.assets)
    .filter(name => cssExtensions.includes(path.extname(name)))
    .map((name) => ({
      size: compilation.assets[name].size(),
      label: name,
    }));
```